client/src/containers/SignIn/SignInForm/SignInForm.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import styled from "styled-components";
+import { Formik, Form, Field } from "formik";
+import FormTextField from "../../../components/FormTextField/FormTextField";
```

Comment:
we should provide some better folder structure. 3 levels up i believe is to much. for now it is ok though.

Reply:
There is a way to use absolute path

`import FormTextField from "components/FormTextField/FormTextField";`